apps/settings/style/apps.css

```diff
@@ -38,7 +38,40 @@
 /**
  * App permissions details panel
  */
-
 .developer-infos > a > small {
   color: #00e;
 }
+
+/*
+ * styles for the addons and addon-details panels
+ */
+.addon-paragraph {
+  font-size: 1.7rem;
+  line-height: 1.7rem;
+  font-weight: 400;
+  font-style: italic;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.addon-paragraph img {
+  float: left;
+  margin: 0 1rem 1rem 1rem;
+  width: 6.4rem;
+  height: 6.4rem;
+}
+
+/******************************************************************************
+ * Right-To-Left layout
+ */
+html[dir="rtl"] #addons ul li img,
+html[dir="rtl"] #addon-details ul li img,
+html[dir="rtl"] #appPermissions ul li img,
+html[dir="rtl"] #homescreens ul li img {
+  left: unset;
+  right: 1.6rem;
+}
+
+html[dir="rtl"] .addon-paragraph img {
```

Comment:
I remembered that there is one mailing thread talking about `[dir="xx"]` and `:dir()` and the latter one is recommended. Can we use `html:dir(rtl)` instead here ? 

Reference : https://developer.mozilla.org/en-US/docs/Web/CSS/:dir